### PR TITLE
fix react-avatar-editor methods

### PR DESCRIPTION
**Pull request by gabsprates**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

---

Commit reference:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/24138/commits/e4aa17e077f28c88abc3f71f9f83c9785fc2242c

The AvatarEditor's methods can't be `static`. It must to refers your own instance.

### CODE REVIEW

File: types/react-avatar-editor/react-avatar-editor-tests.tsx [hunk 1]
```diff
@@ -42,6 +42,6 @@ class AvatarEditorTest extends React.Component {
     }
 }
 
-const getImage: HTMLCanvasElement = AvatarEditor.getImage();
-const getImageScaledToCanvas: HTMLCanvasElement = AvatarEditor.getImageScaledToCanvas();
-const getCroppingRect: CroppedRect = AvatarEditor.getCroppingRect();
+const getImage: HTMLCanvasElement = AvatarEditor.prototype.getImage();
```
Comment (guilhermehubner) [inline types/react-avatar-editor/react-avatar-editor-tests.tsx]: why not `ref` instead `prototype`? :sweat_smile:

Comment (gabsprates) [reply types/react-avatar-editor/react-avatar-editor-tests.tsx]: Well, I also don't like that... I'll try another way.

**Comment by guilhermehubner** [comment]
Nice job @gabsprates, LGTM :wink:  :+1:

**Comment by mosch** [comment]
Thanks! LGTM!

**Pull request merged**